test/Microsoft.AspNet.Mvc.Razor.Test/RazorViewEngineTest.cs

```diff
@@ -73,50 +73,82 @@ public static IEnumerable<object[]> ViewLocationExpanderTestData
         [Theory]
         [InlineData(null)]
         [InlineData("")]
-        public void FindView_ThrowsIfViewNameIsNullOrEmpty(string viewName)
+        public void FindView_IsMainPage_ThrowsIfViewNameIsNullOrEmpty(string viewName)
```

Comment:
We aren't varying the method name based on other parameters (`FindView_ForAreas_WithMainView_Throws` etc). That bit in the name shouldn't be necessary.